hexa/workspaces/templates/workspaces/mails/invite_member.html

```diff
@@ -1,11 +1,11 @@
 {% load i18n %}
 {% autoescape off %}
-{% blocktranslate %}You have been added to the workspace <b>{{workspace}}</b> by {{owner}}.{% endblocktranslate %}
+{% blocktranslate %}You have been invited to join the workspace <b>{{workspace}}</b> by {{owner}}.{% endblocktranslate %}
```

Comment:
IMO, the two mails can be merged (`invite_member` & `invite_external_user`). You could pass the user to the context of the mail to differentiate the two sentences.